src/docker.h

```diff
@@ -60,8 +64,14 @@ class DockerUpdater : public PollingMetadataUpdater {
  public:
   DockerUpdater(MetadataAgent* server)
       : reader_(server->config()), PollingMetadataUpdater(
-          server, server->config().DockerUpdaterIntervalSeconds(),
+          server, 
+          "DockerUpdater", 
```

Comment:
[Optional] You can put this on the previous line.

Reply:
Done.